extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -221,11 +221,23 @@ function _checkIfCanAddMeasurementsToDisplaySet(
     const image = images[i];
     const { SOPInstanceUID } = image;
     if (SOPInstanceUIDs.includes(SOPInstanceUID)) {
-      const imageId = imageIdsForDisplaySet[i];
```

Comment:
Discussed that we will iterate through the imageIds, and use the MetadataProvider.getUIDS... method to get the SOP + frame#

Reply:
Fixed in https://github.com/OHIF/Viewers/pull/2973/commits/75b4cfcef27b972745ba2e3ea3f9546d5f8cf27d